api/app/tests/apps/email_integrations/test_utils.py

```diff
@@ -0,0 +1,172 @@
+# SPDX-License-Identifier: MPL-2.0
+# Copyright (C) 2021 Gemeente Amsterdam
+import re
+
+from django.conf import settings
+from django.test import TestCase
+from faker import Faker
+
+from signals.apps.email_integrations.factories import EmailTemplateFactory
+from signals.apps.email_integrations.models import EmailTemplate
+from signals.apps.email_integrations.utils import (
+    URL_PATTERN,
+    make_email_context,
+    validate_email_template,
+    validate_template
+)
+from signals.apps.signals.factories import SignalFactory
+
+
+class TestUtils(TestCase):
+    def test_regex(self):
+        """
+        Check if a URL matches the URL_PATTERN
+        Check if the URL is replaced by ''
+        Check if the URL is replaced by '' if we put it in a random text
+
+        We test with a couple of examples we can think of and a couple of examples created by the Faker
+        """
+        test_schemas = ['http://', 'https://', 'ftp://', 'sftp://', ]
```

Comment:
There are more, I don't know how far we should go here. But adding `file://`explicitly may be good.

Reply:
I just picked a couple of schemes from https://en.wikipedia.org/wiki/List_of_URI_schemes and added them to the test